specification/communication/data-plane/Rooms/preview/2022-02-01-preview/communicationservicesrooms.json

```diff
@@ -0,0 +1,725 @@
+      "required": [
+        "communicationIdentifier"
+      ],
+      "type": "object",
+      "properties": {
+        "communicationIdentifier": {
+          "$ref": "#/definitions/CommunicationIdentifierModel"
+        },
+        "role": {
+          "$ref": "#/definitions/RoleType"
+        }
+      }
+    },
+    "CommunicationIdentifierModel": {
+      "description": "Identifies a participant in Azure Communication services. A participant is, for example, an Azure communication user. This model must be interpreted as a union: Apart from rawId, at most one further property may be set.",
+      "type": "object",
+      "properties": {
+        "rawId": {
+          "description": "Raw id of the identifier. Optional in requests, required in responses.",
+          "type": "string"
+        },
+        "communicationUser": {
+          "$ref": "#/definitions/CommunicationUserIdentifierModel"
+        }
+      }
+    },
+    "CommunicationUserIdentifierModel": {
+      "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationUserIdentifierModel"
+    },
+    "RoleType": {
+      "description": "The Role of a room participant.",
+      "enum": [
+        "Presenter",
+        "Attendee",
+        "Consumer"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "RoleType",
+        "modelAsString": true
+      }
+    },
+    "RoomJoinPolicy": {
+      "description": "The Policy based on which Participants can join a room.",
+      "enum": [
+        "InviteOnly",
+        "CommunicationServiceUsers"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "RoomJoinPolicy",
+        "modelAsString": true
+      }
+    },
+    "RoomModel": {
+      "description": "The meeting room.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Unique identifier of a room. This id is server generated.",
+          "type": "string",
+          "example": "999126454"
+        },
+        "createdDateTime": {
+          "format": "date-time",
+          "description": "The timestamp when the room was created at the server. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`.",
+          "type": "string",
+          "example": "2021-09-06T05:55:41Z"
+        },
+        "validFrom": {
+          "format": "date-time",
+          "description": "The timestamp from when the room is open for joining. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`.",
+          "type": "string",
+          "example": "2021-09-07T07:55:41Z"
+        },
+        "validUntil": {
+          "format": "date-time",
+          "description": "The timestamp from when the room can no longer be joined. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`.",
+          "type": "string",
+          "example": "2021-09-08T15:55:41Z"
+        },
+        "roomJoinPolicy": {
+          "$ref": "#/definitions/RoomJoinPolicy"
+        },
+        "participants": {
```

Comment:
If we want to prepare ourselves for large collections and keep the separate /participants endpoint, we should not inline participants in the GET rooms/{roomId} response. Also to not have multiple ways of manipulating participants.

Reply:
Captured review comments from Dominik to the user story https://skype.visualstudio.com/SPOOL/_workitems/edit/2930042 to be worked incrementally before GA.